appserver/security/core-ee/src/test/java/com/sun/enterprise/security/ee/perms/SMGlobalPolicyUtilTest.java

```diff
@@ -94,9 +95,10 @@ public void testTYpeConvert() {
         assertThrows(NullPointerException.class, () -> SMGlobalPolicyUtil.convertComponentType(null));
     }
 
-
     @Test
     public void testPolicyLoading() throws NoSuchAlgorithmException, MalformedURLException, URISyntaxException {
+        assumeTrue(System.getProperty("java.vm.specification.version").compareTo("24") < 0);
```

Comment:
Could be better to use this: https://junit.org/junit5/docs/snapshot/user-guide/#writing-tests-conditional-execution-jre